docs/UserGuide.md

```diff
@@ -251,7 +251,89 @@ Examples:
 * `deletetask 2` deletes the task with index 2 from the task list.
 
 
+### Creating a Consult: `createconsult`
+Creates a consultation into upcoming consultations.
+
+Format: `addconsult d/DATE t/TIME n/STUDENT_NAME n/STUDENT_NAME …`
+
+Parameters:
+* DATE The date of consultation
+* TIME The time of consultation
+* STUDENT_NAME The name of the student(s)
+
+Examples:
+* `addconsult d/2023-10-30 t/12:30 n/John Doe` creates a consultation for John Doe on 2023-10-30 12:30.
+* `addconsult d/2023-09-30 t/15:30 n/John Doe n/Foo Bar n/Rayson n/Wesley` creates a consultation for John Doe, Foo Bar
+  , Rayson and Wesley on 2023-09-30 15:30.
+
+
+### Showing list of consultations: `showconsults`
```

Comment:
showconsults and viewconsults may sounds very similar.
Perhaps, you could change showConsults to listConsults, since it lists down all upcoming consultations